packages/pass-style/src/make-far.js

```diff
@@ -128,16 +129,57 @@ export const Remotable = (
 };
 harden(Remotable);
 
+/**
+ * The name of the automatically added default meta-method for obtaining a
+ * list of all methods of an object declared with `Far`, or an object that
+ * inherits from an object declared with `Far`.
+ *
+ * Modeled on `GET_INTERFACE_GUARD` from `@endo/exo`.
+ *
+ * TODO Name to be bikeshed. Perhaps even whether it is a
+ * string or symbol to be bikeshed.
+ *
+ * TODO Beware that an exo's interface can change across an upgrade,
```

Comment:
what is the action TO DO ?

Reply:
Good point. I replaced "TODO" with "HAZARD:". Resolving.